manifests/c/CambridgeUniversityPress/CambridgeReader/1.0210.1/CambridgeUniversityPress.CambridgeReader.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: CambridgeUniversityPress.CambridgeReader
+PackageVersion: 1.0210.1

```

Comment:
```suggestion
PackageVersion: 1.0
```

Reply:
This needs to match Control Panel so `winget upgrade CambridgeUniversityPress.CambridgeReader` doesn't end up in a upgrade loop for version mismatch.

Reply:
You will also need to update the version folder as well.

Change `manifests/c/CambridgeUniversityPress/CambridgeReader/1.0210.1` to `manifests/c/CambridgeUniversityPress/CambridgeReader/1.0`

Reply:
![image](https://user-images.githubusercontent.com/74647769/148465998-c58b196c-5f7c-44a6-9d91-735ea27f47e4.png)